CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 ## 21.6.2
 
 - BREAKING CHANGE: The frontend bundle will be loaded asynchronously (via [#25744](https://github.com/getsentry/sentry/pull/25744)). This is a breaking change that can affect custom plugins that access certain globals in the django template. Please see https://forum.sentry.io/t/breaking-frontend-changes-for-custom-plugins/14184 for more information.
+- feat: Support custom CA roots ([#27062](https://github.com/getsentry/sentry/pull/27062)), see the [docs](https://develop.sentry.dev/self-hosted/custom-ca-roots/) for more details.
```

Comment:
Is this the right place for this entry? Don't we want this in an "Unreleased" section or something? 21.6.2 is already out.

Reply:
Ah, by-product of me not checking what rebase did. Lesson learned.